tests/phpunit/integration/tests/Media/Types.php

```diff
@@ -62,13 +78,22 @@ public function test_get_allowed_mime_types() {
 	 * @covers ::get_allowed_image_mime_types
 	 */
 	public function test_get_allowed_image_mime_types() {
-		$actual   = $this->instance->get_allowed_image_mime_types();
-		$expected = [
-			'image/webp',
-			'image/png',
-			'image/jpeg',
-			'image/gif',
-		];
+		if ( version_compare( get_bloginfo( 'version' ), '5.8.0', '<' ) ) {
```

Comment:
Why are there now these version checks needed to accommodate `image/webp`?

What if some is using WP < 5.8 and uses a plugin to add `image/webp` support?

Can't we detect support this another way?

I'd really, really like to avoid WP version checks and prefer feature detection over this.

Reply:
Changed check.